samples/ar-view-node/src/androidTest/java/com/example/sceneviewsample/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.sceneviewsample
```

Comment:
Should we remove the test files and libraries if they aren't used?

Reply:
fixed